apps/admin-ui/src/component/reservations/ReservationsDataLoader.tsx

```diff
@@ -48,6 +49,14 @@ const mapFilterParams = (
 
   // only use defaults if search is "empty"
   const defaults = emptySearch ? defaultParams : {};
+  // construct a new endDateString, adding one day so the selected date is included
+  const [endDay, endMonth, endYear] = params.end.split(".");
+  const endDateObject = new Date(
+    Number(endYear),
+    Number(endMonth),
+    Number(endDay)
+  );
+  const endDateString = toUIDate(addDays(endDateObject, 1));
```

Comment:
`addDays(endDateObject).toISOString())` to avoid unnecessary `string -> Date` conversions